Owl/gradle/libs.versions.toml

```diff
@@ -50,6 +50,7 @@ secrets = "2.0.1"
 targetSdk = "33"
 
 [libraries]
+accompanist-adaptive = { module = "com.google.accompanist:accompanist-adaptive", version.ref = "accompanist" }
```

Comment:
@alexvanyo is there any reason it's added to all samples?

Reply:
Yup - exactly that, it doesn't getting added to the actual import, just the toml file.